pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -318,6 +318,12 @@ type DashboardSpec struct {
 	// SSL determines whether SSL should be used
 	// +optional
 	SSL bool `json:"ssl,omitempty"`
+	// Prometheus Host end point
+	// +optional
+	PrometheusEndpoint string `json:"prometheusEndpoint,omitempty"`
+	// Prometheus Host end point SSL verify
```

Comment:
Whether to verify the ssl endpoint for prometheus. Set to false for a self-signed cert.